src/EnergyPlus/HVACDXSystem.cc

```diff
@@ -2398,22 +2384,15 @@ namespace HVACDXSystem {
                         // If humidity setpoint is not satisfied and humidity control type is CoolReheat,
                         // then overcool to meet moisture load
 
-                        if ((OutletHumRatDXCoil > DesOutHumRat) && (PartLoadFrac < 1.0) &&
-                            (DXCoolingSystem(DXSystemNum).DehumidControlType == DehumidControl_CoolReheat)) {
+                        if ((OutletHumRatDXCoil > DesOutHumRat) && (DXCoolingSystem(DXSystemNum).DehumidControlType == DehumidControl_CoolReheat)) {
```

Comment:
Now test if outlet node humidity ratio meets humidity ratio set point if CoolReheat control is used.